src/resampling (conflicted copy 2023-02-03 080831).jl

```diff
@@ -0,0 +1,430 @@
+export resample
```

Comment:
not quite sure what you did here but you are causing conflicts with two files.

As a general recommendation, please make small PRs and don't mix bug fixes, features, docs cleaning all together. That makes it easier to review all the changes

Reply:
Probably remove the conflict? file

Reply:
should be gone now